client/app/calendar/tasks/create/page.tsx

```diff
@@ -0,0 +1,43 @@
+'use client'
```

Comment:
I think you don't need calendar layer! Maybe it was miscommunication before we talked. 
`app/tasks/create` is good!

Reply:
Fixed it!
https://github.com/HAK2024/fairy-share/pull/80/commits/e38bca6ce58e8353aaee697e642aab7b88153d95